esp/smc/SMCLib/TpContainer.cpp

```diff
@@ -765,11 +765,11 @@ bool getSashaService(StringBuffer &serviceAddress, const char *serviceName, bool
 {
     if (!isEmptyString(serviceName))
     {
-        VStringBuffer serviceQualifier("services[@type='sasha'][@name='%s']", serviceName);
-        Owned<IPropertyTree> serviceTree = getComponentConfigSP()->getPropTree(serviceQualifier);
+        VStringBuffer serviceQualifier("services[@type='%s']", serviceName);
```

Comment:
I guess we can assume a unique name. Should the @type be @name here?

Reply:
both are generated, type is now the service type (used to be just 'sasha'), see changes in generateConfigMapServices. The scheme only permits the known sasha service types

Reply:
I was confused by the word 'type'. Now, I see that the type is the $sashaName.